packages/fxa-shared/nestjs/health/health.controller.ts

```diff
@@ -30,4 +32,15 @@ export class HealthController {
   versionData(): Version {
     return this.config.version;
   }
+
+  @Get('__exception__')
```

Comment:
First time I've seen `__exception__`, didn't find anything on sentry or nextjs about this endpoint from some quick googling. Do you have a link for more info?

Reply:
That's the URL, so that we can hit `/__exception__` to intentionally cause an exception and verify it gets to Sentry.

Reply:
@bbangert Makes sense, much easier than just adding throw in a random endpoint and hitting it 👍🏽